analyzers/src/SonarAnalyzer.Common/Rules/Utilities/TokenTypeAnalyzerBase.cs

```diff
@@ -30,30 +30,39 @@ namespace SonarAnalyzer.Rules
 {
     public abstract class TokenTypeAnalyzerBase : UtilityAnalyzerBase<TokenTypeInfo>
     {
-        protected const string DiagnosticId = "S9999-token-type";
+        private const string DiagnosticId = "S9999-token-type";
         private const string Title = "Token type calculator";
         private const string TokenTypeFileName = "token-type.pb";
+        private const int IdentifierTokenCountThreshold = 4_000;
```

Comment:
It takes around 3min 21sec on my PC to generate the messages for 4k identifier tokens